spec/services/provider_interface/application_data_export_spec.rb

```diff
@@ -89,7 +89,6 @@ def expect_row_to_match_application_choice(row, application_choice)
         'Award year of degree' => first_degree&.award_year,
         'Institution of degree' => first_degree&.institution_name,
         'Type of international degree' => first_degree&.non_uk_qualification_type,
-        'Equivalency details for international degree' => first_degree&.composite_equivalency_details,
```

Comment:
Does the `composite_equivalency_details` still exist?
I wonder if we have to communicate to whoever uses this if we're removing it 🤔

Reply:
I made a mistake here, we are still using `composite_equivalency_details` https://github.com/DFE-Digital/apply-for-teacher-training/blob/248f163c6aeae08ae1c265ae947118bd7b248ede/app/models/application_qualification.rb#L174

We just need to change the spec here to test for the correct content, previously this was testing the `equivalency_details` but the `composite_equivalency_details` can return more than that.

Reply:
I think the only people we need to notify are the dfe analytics people. Which I'll do after merging but I don't think we should notify anyone using the export or API because we just remove the equivalency details from `composite_equivalency_details` which returns more data than just the `equivalency_details` and there is no `equivalency_details` in production.

Reply:
Can we add this line back in?

Reply:
Not really, this was only checking for `equivalency_details` which was some lorem ipsum string. Now that `equivalency_details` is not part of the `composite_equivalency_details` method it will just return nil and break this spec.

That's why I added a new spec in here https://github.com/DFE-Digital/apply-for-teacher-training/commit/248f163c6aeae08ae1c265ae947118bd7b248ede 

This cannot be done in the old spec because the exporter changes the format of the `composite_equivalency_details` string. It upcases the `Enic` [string](https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/decorators/application_choice_export_decorator.rb#L62). And I can't really control the expect in this [loop](https://github.com/DFE-Digital/apply-for-teacher-training/blob/f9af61b6c0fcdf439b184c7fb4a94fa28914e5a6/spec/services/provider_interface/application_data_export_spec.rb#L103-L105)

Reply:
Is the 'Equivalency details for international degree' row still rendered in the export? If it is can we add a line to ensure that it's still there - even if it is nil/blank 👌